compiler/env/OMRCPU.cpp

```diff
@@ -100,6 +103,12 @@ OMR::CPU::initializeByHostQuery()
 bool
 OMR::CPU::supportsFeature(uint32_t feature)
    {
+   if (TR::Compiler->omrPortLib == NULL)
+      {
+      TR_ASSERT_FATAL(false, "in supports feature without omrPortlib!\n");
+      return false;
+      }
```

Comment:
How about:

```
TR_ASSERT_FATAL(TR::Compiler->omrPortLib != NULL, "NULL portlib pointer encountered while querying feature (%d)\n", feature);
```

and we can avoid the return since the assert is fatal and the return will never execute.